solr/modules/opentelemetry/src/test/org/apache/solr/opentelemetry/BasicAuthIntegrationTracingTest.java

```diff
@@ -31,23 +33,23 @@
 import org.apache.solr.security.BasicAuthPlugin;
 import org.apache.solr.util.SecurityJson;
 import org.apache.solr.util.tracing.TraceUtils;
-import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 public class BasicAuthIntegrationTracingTest extends SolrCloudTestCase {
 
   private static final String COLLECTION = "collection1";
 
+  @ClassRule public static OpenTelemetryRule otelRule = OpenTelemetryRule.create();
```

Comment:
I was about to write an equivalent of OpenTelemetryRule when I found that it already exists :-)

FYI note it has docs that it doesn't work in a ClassRule yet I see it works here.